cluster.js

```diff
@@ -425,7 +432,10 @@ class Cluster {
         this.stats        = {
             active_sessions: 0,
             total_sessions:  0,
-            counter: new StreamCounter()
+            counter: new StreamCounter(),
+            //Counter metric to count total number of requests on a single cluster
+            //separated by 'label: clusterName' from the others
+            cluster_requests_total: cluster_requests_total
```

Comment:
We'll need to collect all metrics relevant to clusters into a separate class or something so that we don't have to add them one-by-one to the objects

Reply:
Now instead of adding them one-by-one we just add a registry that stores all these metrics, you can get the reference of them out of this registry.